concepts/generators/about.md

```diff
@@ -1,2 +1,119 @@
-#TODO: Add about for this concept.
+# About
 
+## Constructing a generator
+
+Generators are constructed much like other functions, but require a [`yield` expression](#the-yield-expression), which we will explore in depth a bit later.
+
+An example is a function that returns the _squares_ from a given list of numbers.  As currently written, all input must be processed before any values can be returned:  
+
+```python
+>>> def squares(list_of_numbers):
+>>>     squares = []
+>>>     for number in list_of_numbers:
+>>>         squares.append(number ** 2)
+>>>     return squares
+```
+
+You can convert that function into a generator like this:
+
+```python
+def squares(list_of_numbers):
+    for number in list_of_number:
+        yield number ** 2
+```
+
+The rationale behind this is that you use a generator when you do not need all the values _at once_.
+
+This would safe you memory, since you store just the value you are _currently_ working on.
+
+## Using a generator
+
+Generators can usually be used in place of most `iterables` in Python. This includes  _functions_ or _objects_ that require an `iterable`/`iterator` as an argument.
+
+For example, if you want to use the `squares()` generator we just constructed, we simply use:

```

Comment:
```suggestion
For example, if you want to use the `squares()` generator we constructed, we use:
```

Reply:
That is even still clunky.  Perhaps `generate we have constructed` is more correct.

I think the problem I am having is why would we make the generator and then decide that we do not want to use it?

So, perhaps:

```suggestion
To use the `squares()` generator:

```

Reply:
Yep, I agree that your last suggestion is probably much clearer! Thanks!